autogen/code_utils.py

```diff
@@ -219,7 +219,7 @@ def _cmd(lang):
     if lang in ["shell"]:
         return "sh"
     if lang in ["ps1"]:
-        return "powershell"
+        return "pwsh"
```

Comment:
Shouldn’t the first condition need to be split to also handle “powershell”?

Reply:
yes youre right, thanks for pointing that out, i have added a split condition to deal with powershell as well could you please review that once as well?

Reply:
I added the pwsh command to the local commandline executor file list of commands as well, please review that once too